cinema_service/settings.py

```diff
@@ -133,6 +133,10 @@
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = BASE_DIR / "media"
+
+MEDIA_URL = "/media/"
```

Comment:
The `MEDIA_ROOT` and `MEDIA_URL` settings are correctly configured to handle media files. Ensure that the media directory is properly secured and accessible in your production environment.